doc/Programming-guide/Key-platform-features/Data-collection/MongoDB-log-appender/index.md

```diff
@@ -0,0 +1,302 @@
+---
+layout: page
+title: MongoDB log appender
+permalink: /:path/
+nav: /:path/Programming-guide/Key-platform-features/Data-collection/MongoDB-log-appender
+sort_idx: 70
+---
+
+{% assign root_url = page.url | split: '/'%}
+{% capture root_url  %} /{{root_url[1]}}/{{root_url[2]}}/{% endcapture %}
+
+* TOC
+{:toc}
+
+The MongoDB log appender is responsible for transferring logs from the Operations service to the MongoDB database. The logs are stored in the table named
+__logs_$[applicationToken]({{root_url}}Glossary)_, where _$[applicationToken]({{root_url}}Glossary)_ matches the token of the current application.
```

Comment:
Due to the underscore in the variable name, the line looks like:
_logs$applicationToken, where _$applicationToken matches ... 

This is incorrect. Try separating the first underscore from the _logs_ or other way(s) to fix the issue.